src/cordova/apple/OutlineAppleLib/Package.swift

```diff
@@ -0,0 +1,45 @@
+// swift-tools-version: 5.6
+// The swift-tools-version declares the minimum version of Swift required to build this package.
+
+import PackageDescription
+
+let package = Package(
+    name: "OutlineAppleLib",
+    products: [
+        .library(
+            name: "OutlineAppleLib",
+            targets: ["Tun2socks", "OutlineTunnel", "OutlineTunnelObjC"]),
+    ],
+    dependencies: [
+        .package(url: "https://github.com/CocoaLumberjack/CocoaLumberjack.git", exact: "3.7.4"),
+    ],
+    targets: [
+        .target(
+            name: "OutlineTunnelObjC",
+            dependencies:
+                ["CocoaLumberjack",
+                 .product(name: "CocoaLumberjackSwift", package: "CocoaLumberjack"),
+                 "Tun2socks",
+                 "OutlineTunnel"
+                ],
+            path: "./Sources/OutlineTunnelSources/",
+            exclude: ["SwiftSources"],
```

Comment:
Any reason why the swift code for OutlineTunnel is nested under the PacketTunnelProvider directory?